packages/preview-middleware/package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "@sap-ux/preview-middleware",
+    "description": "Preview middleware",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/preview-middleware"
+    },
+    "bugs": {
+        "url": "https://github.com/SAP/open-ux-tools/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Apreview-middleware"
+    },
+    "version": "0.0.1",
+    "license": "Apache-2.0",
+    "author": "@SAP/ux-tools-team",
+    "main": "dist/index.js",
+    "scripts": {
+        "start:fixture": "ui5 serve --config test/fixtures/simple-app/ui5.yaml",
+        "build": "tsc --build",
+        "watch": "tsc --watch",
+        "clean": "rimraf dist test/test-output coverage *.tsbuildinfo",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "ui5.yaml",
+        "!dist/*.map",
+        "!dist/**/*.map"
+    ],
```

Comment:
We need to add `templates` otherwise the templates will not be part of the package.

```
Error: ENOENT: no such file or directory, open '/projects/GTP/v4app/node_modules/@sap-ux/preview-middleware/templates/flp/sandbox.html'
    at Object.openSync (node:fs:590:3)
    at readFileSync (node:fs:458:35)
    at /projects/GTP/v4app/node_modules/@sap-ux/preview-middleware/dist/base/flp.js:79:52
    at Layer.handle [as handle_request] (/projects/GTP/v4app/node_modules/express/lib/router/layer.js:95:5)
    at next (/projects/GTP/v4app/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/projects/GTP/v4app/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/projects/GTP/v4app/node_modules/express/lib/router/layer.js:95:5)
    at /projects/GTP/v4app/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/projects/GTP/v4app/node_modules/express/lib/router/index.js:341:12)
    at next (/projects/GTP/v4app/node_modules/express/lib/router/index.js:275:10)
```

Reply:
https://github.com/SAP/open-ux-tools/pull/1093/commits/81845611f1c9d1a17e6eeb26aab051f65c0531ce

Reply:
thx